pennylane/devices/qubit/apply_operation.py

```diff
@@ -228,9 +229,23 @@ def _(op: type_op, state):
         [1., 0.]], requires_grad=True)
 
     """
+    if sp.sparse.issparse(op.matrix()):
+        return apply_operation_csr_matrix(op, state, is_state_batched)
     return _apply_operation_default(op, state, is_state_batched, debugger)
 
 
+def apply_operation_csr_matrix(op, state, is_state_batched: bool = False):
+    """The csr_matrix specialized version apply operation."""
+    # State is numpy array, should have been stored in tensor version
+    # remember the initial shape and recover in the end
+    original_shape = math.shape(state)
+    num_wires = len(original_shape) - is_state_batched
```

Comment:
```suggestion
    num_wires = len(original_shape) - int(is_state_batched)
```

Although it's probably unnecessary, explicitly casting to `int` might clarify this. What do you think?

Reply:
LGTM. I wrote that because it's been used almost everywhere in DQ (also in default mixed because I followed DQ style). We can even have a backlog for it to explicit cast all these implicit arithmetic usage of booleans.